src/components/RecipesForm.js

```diff
@@ -0,0 +1,114 @@
+import { useState } from "react"
+import { TextField, Button } from "@mui/material"
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+import '../styles/RecipesForm.css'
+
+const RecipesForm = ({reloadList, setReloadList}) => {
+    
+    const [recipeName, setRecipeName] = useState('')
+    const [cuisineType, setCuisineType] = useState('')
+    const [estTimeOfPrep, setEstTimeOfPrep] = useState('')
+    const [ingredients, setIngredients] = useState('')
+    const [prepInstructions, setPrepInstructions] = useState('')
+
+    const handleRecipesForm = (event) => {
+        event.preventDefault()
+        if (recipeName && cuisineType && estTimeOfPrep && ingredients && prepInstructions) {
+            const recipeObject = {
+                recipeName,
+                cuisineType,
+                estTimeOfPrep,
+                ingredients,
+                prepInstructions
+            }
+            fetchCreateRecipe(recipeObject)
+        }
+    }
+
+    const handleChange = (event) => {
+        setCuisineType(event.target.value);
+    };
+    
+    const fetchCreateRecipe = async (recipe) => {
+
+        const token = localStorage.getItem('token')
+        const userId = localStorage.getItem('user')
+
+        const url = `https://recipe-blog-l7ey.onrender.com/recipe-blog/mypage`
+
+        const options = {
+            method: 'POST',
+            headers: {
+                Authorization:`Bearer ${token}`,
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+                "recipeName": recipe.recipeName,
+                "createdBy": userId,
+                "cuisineType": recipe.cuisineType,
+                "estTimeOfPrep": recipe.estTimeOfPrep,
+                "ingredients": recipe.ingredients,
+                "prepInstructions": recipe.prepInstructions
+            })
+        }
+
+        try {
+            const response = await fetch(url, options)
+
+            if (!response.ok) {
+                alert(response.status)
+                throw new Error(`Error: ${response.status}`)
+            }
+
+            const data = await response.json()
+
+            console.log(data)
+
+            
+            setReloadList(reloadList + 1)
+        } catch (error) {
+            console.log(error)
+        }
+    }
+    
+    
+    return (
+        <>
```

Comment:
Add the following code to display an error message if the error state is not null:
```
{errorMessage !== null && (
  <div>
    <h3 style={{textAlign: 'center', color: '#F00' }}>{errorMessage}</h3>
  </div>
)}
```